stream-chat-android-ui-components/src/main/res/values/attrs_channel_list_view.xml

```diff
@@ -60,7 +60,9 @@
 
         <attr name="streamUiUnreadMessageCounterBackgroundColor" format="color|reference" />
 
-        <attr name="streamUiMutedChannelIcon" format="reference"/>
-        <attr name="streamUiMutedChannelIconTint" format="color|reference"/>
+        <attr name="streamUiMutedChannelIcon" format="reference" />
+        <attr name="streamUiMutedChannelIconTint" format="color|reference" />
+
+        <attr name="streamUiChannelActionsDialogStyle" />
     </declare-styleable>
```

Comment:
Make `streamUiChannelActionsDialogStyle` attribute to reference an overriden style to override channel actions style for this particular `ChannelListView`